spec.html

```diff
@@ -9039,7 +9039,7 @@ <h1>FunctionDeclarationInstantiation ( _func_, _argumentsList_ )</h1>
           1. Set the LexicalEnvironment of _calleeContext_ to _env_.
         1. For each String _paramName_ in _parameterNames_, do
           1. Let _alreadyDeclared_ be _env_.HasBinding(_paramName_).
-          1. NOTE: Early errors ensure that duplicate parameter names can only occur in non-strict functions that do not have parameter default values or rest parameters.
+          1. NOTE: Early errors ensure that duplicate parameter names can only occur in non-strict functions that do not have parameter default values or <emu-not-ref>rest parameters</emu-not-ref>.
```

Comment:
I'd prefer we just not `dfn` this term, rather than `emu-not-ref`'ing half of its uses.